### PR TITLE
Add Sennheiser

### LINKED ISSUE #3016: Sennheiser logo request
**Name: Sennheiser**
**Website: https://sennheiser.com/**
**Official resources for icon and color: https://de.wikipedia.org/wiki/Sennheiser#/media/Datei:Sennheiser_logo.svg http://logok.org/sennheiser-logo/**
  

Add Sennheiser #3045

### PR DESCRIPTION
**Pull request by Orchomenos**
sennheiser

**Issue:** Closes #3016

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by ericcornelissen** [comment]
From [the build error](https://travis-ci.com/github/simple-icons/simple-icons/jobs/330538139#L232) it seems that there is a `class` attribute on the `<path>` of the SVG, could you remove that. Thanks :slightly_smiling_face: 

From the commits listed here, it looks like you have branched off of `master` rather than `develop`. Could that be correct? Next time you try to branch, make sure that you create the new branch when you're on the `develop` branch, that keeps things clean. (any questions?)  - For now this is fine, as we will `Squash and merge` this Pull Request anyway, and hence the redundant commits won't end up in the repository history. (@simple-icons/maintainers, when merging this make sure to remove the commit messages of the release commits)

**Comment by Orchomenos** [comment]
> From [the build error](https://travis-ci.com/github/simple-icons/simple-icons/jobs/330538139#L232) it seems that there is a `class` attribute on the `<path>` of the SVG, could you remove that. Thanks 🙂
> 
> From the commits listed here, it looks like you have branched off of `master` rather than `develop`. Could that be correct? Next time you try to branch, make sure that you create the new branch when you're on the `develop` branch, that keeps things clean. (any questions?) - For now this is fine, as we will `Squash and merge` this Pull Request anyway, and hence the redundant commits won't end up in the repository history. (@simple-icons/maintainers, when merging this make sure to remove the commit messages of the release commits)

Thanks for the tipp, the JBL Harman logo should be a seperate branch from my default branch now. I also fixed the problem with the weird class in the Sennheiser logo.

**Comment by ericcornelissen** [comment]
> Thanks for the tipp, the JBL Harman logo should be a seperate branch from my default branch now. I also fixed the problem with the weird class in the Sennheiser logo.

Thanks :+1: 


> now using this version of the logo: https://de-de.sennheiser.com/assets/v4r/syms-9fb6701df6cf24b2cec777746b2005557baa25d21c5e3b60663524b4e3a241df.svg

Where did you find this URL, what web page? If it is the homepage, [as pointed out in the issue](https://sennheiser.com/), then could you update the source URL to just `https://en-fi.sennheiser.com/`. If not, can you share the page where you found it here? (the current source URL is very likely to change).

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5225,6 +5225,11 @@
             "hex": "86BC40",
             "source": "http://design.sencha.com/"
         },
+        {
+            "title": "Sennheiser",
+            "hex": "000000",
+            "source": "https://de-de.sennheiser.com/assets/v4r/syms-9fb6701df6cf24b2cec777746b2005557baa25d21c5e3b60663524b4e3a241df.svg"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:5231]:
```suggestion
            "source": "https://sennheiser.com"
```

**Pull request merged**